### DISCUSSION TITLE
Welcome to Otter-Grader Discussions!

### DISCUSSION THREAD

**Discussion by chrispyles** [ucbds-infra/otter-grader#841] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

This forum is for public discussion and questions. We also have a Slack that you can join:

[![Slack](https://img.shields.io/endpoint?logo=slack&url=https%3A%2F%2Fraw.githubusercontent.com%2Fucbds-infra%2Fotter-grader%2Fmaster%2Fslack-shields.json)](https://join.slack.com/t/otter-grader/shared_invite/enQtOTM5MTQ0MzkwMTk0LTBiNWIzZTYxNDA2NDZmM2JkMzcwZjA4YWViNDM4ZTgyNDVhNDgwOTQ0NjNlZjcwNmY5YzJiZjZhZGNhNzc5MjA)

**Comment by RWillisConger** [comment]


**Reply by chrispyles** [reply]
You can try the steps here to debug the submission: https://otter-grader.readthedocs.io/en/latest/debugging.html

Other things to look at are package version mismatches or errors in Gradescope's image build output.

**Reply by RWillisConger** [reply]
Thank you!
